packages/docs/README.md

```diff
@@ -1,60 +1,181 @@
 
-Punctual problems in translations, such as spelling errors or confusing phrases, can be reported by sending _issues_ in this repository. Make it clear which page the problem is referring to, and preferably offer a suggestion to improve clarity.
+### Commit types
+* **feat:** Commits that result in a new features or functionalities. Backwards compatible features will release with the next **MINOR** whereas breaking changes will be in the next **MAJOR**. The body of a commit with `breaking changes` must begin with BREAKING CHANGE, followed by a description of how the API has changed.
+* **fix:** Commits that provide fixes for bugs within vuetify's codebase.
+* **docs:** Commits that provide updates to the docs.
+* **style:** Commits that do not affect how the code runs, these are simply changes to formatting.
+* **refactor:** Commits that neither fixes a bug nor adds a feature.
+* **perf:** Commits that improve performance.
+* **test:** Commits that add missing or correct existing tests.
+* **chore:** Other commits that dont modify src or test files.
+* **revert:** Commits that revert previous commits.
 
-If you feel comfortable translating/reviewing content on your own, the steps are:
+## Submitting Changes / Pull Requests
+Before doing any commits, you will want to pull down the latest and greatest from dev. From here, merge, and resolve any conflicts between your branch and dev. Its a good rule of thumb to pull frequently as development is constantly happening.   
 
-- Create an _issue_ in this _repo_, of type `work-in-progress`, to inform us of what you are doing
-- Make an _fork_ of this _repo_ to your own account
-- Run `yarn` and then `yarn translate`
-- Go to http://localhost:8095, and either select the language you want to work on from the dropdown menu, or create a new one using the _New translation_ option
-- You can now translate/revise the content directly on the site using the _pen_ icons
-- When finished, make a _pull request_ **with the _commit_ description in English**
-- All PRs should be made to the `dev` branch
-- Do not forget to `pull` the latest changes before resuming the process.
+In git, add all relevent files.
 
-If you can not completely finish a translation, but want to send the partial work, make it clear in the _issue_ of your work after the submission.
+Commit with commitizen using the command `git cz`. From here you will follow through a series of props. Make sure to select the appropiate type (see **Commit Guidelines w/Commitizen** above)
+
+Lastly, `git push` and open a pull request.
+
+### Pull Requests For Vuetify
+
+> Pull requests related to Vuetify: for bug fixes submit pull requests to `master`, for new features and breaking changes submit pull requests to `dev`.
```

Comment:
breaking changes should go to `next`

Reply:
![image](https://user-images.githubusercontent.com/16918043/61996551-6eb62000-b0a6-11e9-9e24-a41b278d6656.png)

Reply:
here and there then need?
![image](https://user-images.githubusercontent.com/16918043/61996662-c4d79300-b0a7-11e9-839b-26c5d8e6cc2a.png)

Reply:
Yeah probably should update the contribution page as well

Pull requests related to Vuetify: 
* For **bug fixes** and **documentation updates** submit pull requests to `master`.
* For **new features** and **enhancements** submit pull requests to `dev`
* For any **features** that contain **breaking changes** submit pull requests to `next`